pyschism/forcing/hycom/hycom2schism.py

```diff
@@ -72,6 +76,9 @@ def get_idxs(date, database, bbox):
     
     lon=ds['lon'][:]
     lat=ds['lat'][:]
+
```

Comment:
If you need to modify xmin, xmax, this would be a better place to do it. Maybe check the values of lon lat to check if you need to convert?